EEPS/eep-4.md

```diff
@@ -0,0 +1,277 @@
+is not being proposed as an actionable vote for Block Producers. This should only be used for polling sentiment.
+The available responses are always `0`:`No`, `1`:`Yes`
+All UIs should display the voting buttons along with their value e.g. "0 - No" and "1 - Yes"
+so that a user can easily verify their vote through any block explorer, without the need to already
+have the knowledge that a `vote_value` of 0 properly signifies their vote of "No", and so on.
+
+```
+{
+  "type":"poll-yn-v1",
+  "content":"content here"
+}
+```
+
+Example:
+```
+{
+  "type":"poll-yn-v1",
+  "content":"Should EOS change its symbol to SYS?"
+}
+```
+
+**_Summary Notes_**
+
+Standard `vote_value`: 
+
+`0` - "No" 
+
+`1` - "Yes" 
+
+As this `type` of poll is not used for an official chain-altering decision, Block Producers
+should **not** act in any meaningful way to the results of these polls. This can be useful for 
+determining a path to take in further discussions, or whether or not a question would be worthwhile
+being proposed for a `referendum`. 
+
+### Yes/No/Abstain Poll
+
+`poll-yna-v1` should be used for any polling that uses only Yes/No/Abstain responses, and 
+is not being proposed as an actionable vote for Block Producers. This should only be used for polling sentiment.
+The available responses are always `0`:`No`, `1`:`Yes`, `2`:`Abstain`
+All UIs should display the voting buttons along with their value e.g. "0 - No", "1 - Yes", and "2 - Abstain"
+so that a user can easily verify their vote through any block explorer, without the need to already
+have the knowledge that a `vote_value` of 0 properly signifies their vote of "No", and so on.
+
+_For clarification, abstain is defined as "formally decline to vote either for or against a proposal or motion"_
+
+```
+{
+  "type":"poll-yna-v1",
+  "content":"content here"
+}
+```
+
+Example:
+```
+{
+  "type":"poll-yna-v1",
+  "content":"Should EOS change its symbol to SYS?"
+}
+```
+
+**_Summary Notes_**
+
+Standard `vote_value`: 
+
+`0` - "No" 
+
+`1` - "Yes" 
+
+`2` - "Abstain"
+
+As this `type` of poll is not used for an official chain-altering decision, Block Producers
+should **not** act in any meaningful way to the results of these polls. This can be useful for 
+determining a path to take in further discussions, or whether or not a question would be worthwhile
+being proposed for a `referendum`. 
+
+The addition of "Abstain" allows a user to have their vote count towards participation
+to help signal community enthusiasm, while essentially delegating their decision to those 
+who cast a Yes or No vote. This can help for deciding if a vote should then become a
+referendum, and whether or not it might receive enough voter participation.
+
+### Options Poll
+
+`options-v1` should be used for any polling that requires multiple custom responses (`proposer` can
+specify up to 255 different responses, always leaving one option for `Abstain`). UIs should fetch the 
```

Comment:
What would be the `vote_value` for `Abstain`? Should we make it part of the standard right away? `255` maybe since it's probably the value that has least chance of being used by "mistake".